Switch to OCA maintainer tools

Cedric-Pigeon opened the pull request.

Commit:
[CHG] switch to OCA maintainer-quality-tools

Commit:
[CHG] switch to OCA maintainer-quality-tools

sbidoul commented on .travis.yml:
```diff
@@ -38,15 +38,11 @@ env:
   - printf '[options]\n\nrunning_env = dev\nkeychain_key_dev = HRdthrkXp28SCHX0kmwOgIob-82N-w8zWoE_UsctE7E=' > ${HOME}/.openerp_serverrc
-  - git clone https://github.com/akretion/akretion-quality-tools ${HOME}/akretion-quality-tools
-  - export PATH=${HOME}/akretion-quality-tools/travis:${PATH}
-  - export PATH=.travis:${PATH}
```
This `.travis` in PATH must stay, because that is where `travis_delete_algolia_index` is.

sbidoul committed:
add missing encoding headers

sbidoul committed:
replace deprecated description field in manifests

sbidoul committed:
[FIX] add missing translation

sbidoul committed:
ignore false positive pylint warning

sbidoul committed:
temporary disable pylint warning (TODO)

sbidoul committed:
run pylint in pre-commit

sbidoul committed:
pre-commit fix-encoding-pragma

sbidoul committed:
add pip and pre-commit cache

sbidoul committed:
Install dependencies with pip

sbidoul committed:
fix deprecated git requirement form

sbidoul committed:
fix external dependency python-magic

sbidoul committed:
we need cachetools<3

sbidoul committed:
add TODO

sbidoul committed:
add test requirements

They where previously installed by dependent repos.

Before they where added by a dependent repo.

lmignon commented:
Great!

Cedric-Pigeon commented:
/ocabot merge patch
